client/Main.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+
+export default class Main extends React.Component {
+  constructor(){
+    super()
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.handleChange = this.handleChange.bind(this)
+    this.addEducation = this.addEducation.bind(this)
+    this.state = {
+      fullName: "",
+      title: "",
+      email: "",
+      education: [{institution:""
+      // startDate: "",
+      // endDate:""
+      }]
+    }
+  }
+
+  handleSubmit(event) {
+    event.preventDefault()
+    console.log(this.state)
+  }
+
+  handleChange(event) {
+    if (["institution"].includes(event.target.className)){
+      let education = [...this.state.education]
+      education[event.target.dataset.id][event.target.className] = event.target.value
+      this.setState({education}, () => console.log(this.state.education))
+    } else {
+      this.setState({[event.target.name]: event.target.value})
+    }
+  }
+
+  addEducation(event){
+    this.setState((prevState) => ({
+      education: [...prevState.education, {institution:""}]
+    }))
+  }
+
+  render () {
+    let {education} = this.state
+    return (
+      <form className = "basic-info" onSubmit={this.handleSubmit} onChange={this.handleChange}>
+        <label htmlFor="fullName">Full Name</label>
+        <input
+          name="fullName"
+          value={this.state.fullName}
```

Comment:
fullName can be included in your de-structuring code on line 42. I would stay consistent with line 42 and include the below state values in it.